docs/site/Server.md

```diff
@@ -338,6 +338,51 @@ export class HelloWorldApp extends Application {
 You can also add multiple servers in the constructor of your application class
 as shown [here](Application.md#servers).
 
+### Enhance OpenAPI Specification
+
+The rest server exposes a function `getApiSpec()` to retrieve its OpenAPI
+specifications:
+
+```ts
+// in code, retrieve the OpenAPI spec by `getApiSpec()`
+const spec = await app.restServer.getApiSpec();
+```
+
+An application's OAS is mainly generated from
+[controllers](https://loopback.io/doc/en/lb4/Controllers.html) and their
+members' metadata. Besides this, we would also like to contribute specifications
```

Comment:
Do you mean:
we also allow other metadata to be added to the OpenAPI spec?

Reply:
I am trying to say **other than controller**, the spec can be contributed from other places :)

Reply:
rephrased it to 
> An application's OpenAPI specification is mainly generated from
[controllers](https://loopback.io/doc/en/lb4/Controllers.html) and their
members. Besides the controller, other artifacts should also be able to
contribute specifications.